### PR TITLE
Uninstall the extension when the time elapses.

### PR DESCRIPTION
**Pull request by msramek**
A heartbeat function runs periodically to check whether the extension has a given number of days of uptime or if a static deadline has been reached.

### CODE REVIEW

**Comment by msramek** [comment]
This is a very similar mechanism as we had with maybeShowNotification. It's probably still up to discussion if we want to do it this way, but I believe we want to do at least a subset of this.

File: chrome/sunset.js [hunk 1]
```diff
@@ -163,12 +182,59 @@ Sunset.showArticle_ = function() {
...
+}
+
+
+/**
  * Triggers a loop that tests if conditions were met to show the message.
  */
 Sunset.run = function() {
   chrome.browserAction.onClicked.addListener(Sunset.showArticle_);
   chrome.notifications.onButtonClicked.addListener(Sunset.showLandingPage_);
-  chrome.alarms.onAlarm.addListener(Sunset.maybeShowNotification_);
```
Comment (mikewest) [inline chrome/sunset.js]: Can we remove this method entirely?

**Comment by mikewest** [comment]
Looks pretty reasonable to me. LGTM.

File: chrome/sunset.js (hunk 1 above)
Comment (msramek) [inline chrome/sunset.js]: Yes, and several other methods and attributes that were only referenced by it (I wanted to do a large cleanup CL in the end, but we can also do this now).

**Pull request merged**